src/api/app/views/webui2/webui/configuration/index.html.haml

```diff
@@ -3,4 +3,38 @@
 
 .card.mb-3
   = render partial: 'tabs'
-  .card-body
+  .card-body#configuration
+    %h3.mb-4 Configure your Open Build Service instance
+    = form_for(@configuration, url: { action: :update }) do |f|
+      .row
+        .col-xl-6.mb-2
+          .h5 Frontpage Greeting
+          %p.text-muted
+            Change the text that is displayed as greeting on the frontpage
+          .form-group
+            = f.label :title
+            = f.text_field :title, required: true, class: 'form-control'
+          .form-group
+            = f.label :description
+            ~ f.text_area :description, rows: 7, required: true, class: 'form-control'
+        .col-xl-6.mb-2
+          .h5 Base Configuration
+          %p.text-muted
+            %i.fas.fa-exclamation-circle.text-warning
```

Comment:
It should be `fa-exclamation-triangle` according with `text-warning` according to our [patterns library](https://obs-patterns.netlify.com/atoms/icons.html)